config/environments/development.rb

```diff
@@ -9,9 +9,15 @@
   ENV['USE_PLACEHOLDER_STUDENT_PHOTO'] = 'true'
   ENV['USE_PLACEHOLDER_IEP_DOCUMENT'] = 'true'
   ENV['CONSISTENT_TIMING_FOR_LOGIN_IN_MILLISECONDS'] = '2000'
+  # For use with studentinsights-dev Google account
+  ENV['GOOGLE_PRIVATE_KEY'] = '-----BEGIN PRIVATE KEY-----\nMIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQCKYeWIf90xpTPI\nw2RfF2ZiB4W9ObBt78oI0IWmdBtcjnYgkoRWkis/RWPEC8hvr2qupLG+CB6PaFKX\niNq0p9IE+8MYiAD54A1vZu9QO60Klx2xzt+p/RMqxhjiDqzXjjdvwXDg4EsuM0PC\nfx01+HcR7qeoG6So2VIqpZVHsrfat7BKVZnSQkiY+1/wKTNm5QTKHOnruqfu2312\nLqndTGyJX+ijhmmIEDn/wRv05xRVc8E3ydaz10QwdFx+LxmxUld4b5ZrlTOo2UHq\nLlllvf/iSGrtSva/3LRzvV4qi3GrFymyceB5GbnUKMIJcU8JR9mwwHFIvQ57hAvC\nSp0+UE5fAgMBAAECggEAQNpkcjZKecvMmxhHX/FdgC+6U9ZHgx5P9xWSrVHYyOXT\nIqzH7LKQJhTlDqnfA8iYHMT+r+qqN9dSbcYriWLlIITFCCglHPYOFdmN6nItRLtR\npVUoqUf7xZ4VZ0wtYNA71to3hoxXb42DAhIEpJiPZQ4N8NETuDYxNd9oOLPfSlb/\ngJSsZ/ZOMB7WZ1RH7RkumB6kohkZkrbB+7z7OQMuDk3fB4McIrcMH3FIc9zXu8iR\nWVal8g+e156fBt+zEr+lV+gIBpUu18nlRfUv73xFXW/axhdjeKsxgchs7IlbpRI4\n99VwOC4PyYEkp1CU0pDd/X2qzkyP6oGjAaXTPnBTUQKBgQDBkDVLWGd5W77exFpR\nPSZp0RDtDi719gKl8I5aO9eiKgAuqexsaeEBhmztvLRtFPOojVEgo7aiahzcJJRL\nULbsGPtrcmCE8PDotFtmp+RRcMfBzu+4/vvTtmR2wGDby19FfLD8hqwYfAxSNeQe\nV1pFMzzl0OKQ3PscJljMBSlu0wKBgQC3BQoTyuXgif4nmiyKkVsFlTNKBjj3EghU\nJGYxEyRIVbS7YWMly0ImkP0H89cjiaU+geWfK1YeQWMQnaDsqbLfjJ4o84cnRGVD\nlzBvHDvTq3QoCzfN+yroVT6CdCHy6/0N+/0ilD2njIwB3iSvcY0oFTJCgRPESnSe\nkOKwsrIixQKBgDJecl/e6147L5oMgmwte3eBvePSEwpSMDI6PPDiLaS3RUDfWvSD\nQNgCwQnEVzcjgpxT2c6ii34MGuJ2aliG8nLRtWsqYk9121rSxA4cll9S98hJLzx2\nrdVT2yTDvvCzJWGRpwSvjsz4SsSB3ZC/rm/fe3NaPfrwZi/XErhUh4mdAoGAIoIj\nvUmjgGvkbQkn3tS7adkQ+6MiuItA4MSgNB6c17q5Qh+Bt02U4nqyCv7SENRSQO/z\nIMjmCvxfNnHSNu1kWbxE0gng8Ol3goAOVNTo/uc0FpzO39q842O3TpehExGl/C4I\nM63p1h6tA2kkV6Q8blNYz6tQ09EKHh/JbnGzHtUCgYA/BkL+WiXs3nSlyFmJwmmJ\nyATwCiKbTZXzVTBQgxgSd9+OlWTYEcQHDp6TYIHM9bDpgluMxVH9ctRPvt44cS1a\n0Y7BjAV6VMJjIE5VtOhYN5pHQZAglt4wE8b5NvWA8/E6/KXVErOQBbmzaizddk2z\ncmGatg2+i//aRP92+8lwWQ==\n-----END PRIVATE KEY-----\n'
+  ENV['GOOGLE_CLIENT_EMAIL'] = 'student-insights-dev@student-insights-dev.iam.gserviceaccount.com'
+  ENV['GOOGLE_PROJECT_ID'] = '102517779824220641067'
+  ENV['GOOGLE_ACCOUNT_TYPE'] = 'service_account'
```

Comment:
What do you think about moving this into one variable, like `GOOGLE_SHEETS_IMPORTER_JSON`?  Then the importer class can grab what it needs out of their and update the ENV before running the code.

```
GOOGLE_SHEETS_SYNC_CREDENTIALS_JSON='{...}'
json = JSON.parse(ENV['GOOGLE_SHEETS_SYNC_CREDENTIALS_JSON'])
ENV['GOOGLE_ACCOUNT_TYPE'] = json['type']
ENV['GOOGLE_CLIENT_EMAIL'] = json['client_email']
ENV['GOOGLE_CLIENT_ID'] = json['client_id']
ENV['GOOGLE_PRIVATE_KEY'] = json['private_key']
```

Reply:
In theory that should work! In practice I ran into trouble when I didn't set these explicitly, but I think that's likely a local configuration problem.